history-of-quakers/modernized/12-chapter-10.adoc

```diff
@@ -1096,27 +1201,38 @@ to do and receive those things which his Majesty`'s said
 justice shall determine concerning him in this behalf,
 as by the aforesaid order is required.
 
-GEORGE CHETHAM, Esq.
-Sheriff.
+[.signed-section-signature]
+George Chetham, Esq. Sheriff.
+
+--
 
 The king upon perusal of this, and consideration of the whole matter,
 being satisfied of G. Fox`'s innocency,
 commanded his secretary to send the following order to judge Mallet for his release:
 
+[.embedded-content-document.legal]
+--
+
 It is his Majesty`'s pleasure that you give order for the releasing,
 and setting at full liberty, the person of George Fox, late a prisoner in Lancaster jail,
-and commanded here by an habeas corpus.
+and commanded here by an _habeas corpus._
 And this signification of his Majesty`'s pleasure shall be your sufficient warrant.
 Dated at Whitehall, the 24th of October, 1660.
 
-EDWARD NICHOLAS.
+[.signed-section-signature]
+Edward Nicholas.
 
 For Sir Thomas Mallet, knight, one of the justices of the King`'s Bench.
```

Comment:
I had put two of these things in [sscc] in the original and thought I had done it here too. Idk if it's the right thing to do, but it looked a little better in the PDF. I'll mark the other one as well, it's just below this one.

Reply:
yeah, i think it should be a [sscc] for this and the one below, let's go with how you did it in the original.